test/e2e/node_drain_test.go

```diff
@@ -291,15 +295,19 @@ func generateStatefulset(input generateStatefulsetInput) *appsv1.StatefulSet {
 		Spec: appsv1.StatefulSetSpec{
 			Selector: &metav1.LabelSelector{
 				MatchLabels: map[string]string{
-					"app":         "nonstop",
-					"statefulset": input.Name,
+					"app":               "nonstop",
+					"e2e-test":          "node-drain",
+					boskosResourceLabel: os.Getenv("BOSKOS_RESOURCE_NAME"),
```

Comment:
What if BOSKOS_RESOURCE_NAME is not set? Do we need a fallback?

Reply:
not set should only be the case when running manually (and not setting it).

I considered not adding it, but thought: better to have an empty label to identify these pvc's at the end.